src/test/java/org/mifos/integrationtest/cucumber/stepdef/CollectionPaymentSchemeDef.java

```diff
@@ -0,0 +1,25 @@
+package org.mifos.integrationtest.cucumber.stepdef;
+
+import io.cucumber.java.en.When;
+import io.restassured.RestAssured;
+import io.restassured.builder.ResponseSpecBuilder;
+import io.restassured.specification.RequestSpecification;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.mifos.integrationtest.common.CollectionHelper;
+import org.mifos.integrationtest.common.Utils;
+
+public class CollectionPaymentSchemeDef extends BaseStepDef {
+
+    @When("I call the collection endpoint with payment scheme {string} and status is {int}")
+    public void iCallTheCollectionEndpointWithPaymentSchemeAndExpectedStatus(String paymentScheme, int expectedStatus)
+            throws JSONException {
+        RequestSpecification requestSpec = Utils.getDefaultSpec(BaseStepDef.tenant);
+        requestSpec.header(Utils.X_CORRELATIONID, BaseStepDef.clientCorrelationId);
+        requestSpec.header(Utils.PAYMENT_SCHEME_HEADER, paymentScheme);
+        JSONObject collectionRequestBody = CollectionHelper.getCollectionRequestBody("1", "254708374149", "24450523");
```

Comment:
Why are these values hardcoded, You have to add steps to create this data.

Reply:
Yeah, that would make sense. But I was only following what I've seen in all the existing tests. For example, this one - https://github.com/openMF/ph-ee-integration-test/blob/5ad36c461f6e5ef22d902a264f307efcac6355a9/src/test/java/org/mifos/integrationtest/cucumber/stepdef/IdempotencyStepDef.java#L36

Reply:
we should be trying to improve the issues we have in existing test cases. There have been many quality issues when CI was not running these tests.
The quality of the test case will improve only when each test case is independent. Otherwise these test will endup low quality and flaky.